src/Tribe/Status/Abstract_Commerce.php

```diff
@@ -0,0 +1,281 @@
+
+		$sold = $ticket_sold['completed'] ? $ticket_sold['completed'] : $ticket_sold['sold'];
+
+		$sold_message = ! $ticket_sold['has_stock'] ?
+			$sold_message = sprintf( __( 'Sold %d', 'event-tickets-plus' ), esc_html( $sold ) ) :
+			'';
+
+		$price = $ticket_sold['ticket']->price ?
+			' (' . tribe_format_currency( number_format( $ticket_sold['ticket']->price, 2 ), $post_id ) . ')' :
+			'';
+
+		$sku = $ticket_sold['sku'] ?
+				'title="' . sprintf( esc_html__( 'SKU: (%s)', 'event-tickets-plus' ), esc_html( $ticket_sold['sku'] ) ) . '"' :
+				'';
+
+		ob_start();
+		?>
+		<strong <?php echo $sku; ?>><?php echo esc_html( $ticket_sold['ticket']->name . $price ); ?>:</strong>
+		<?php
+		echo esc_html( $sold_message );
+
+		return ob_get_clean();
+
+	}
+
+	/**
+	 * Get the Available and Incomplete Counts for a Ticket
+	 *
+	 * @since tbd
+	 *
+	 * @param $ticket_sold object an object of the ticket to get counts
+	 *
+	 * @return bool|string a string of available and/or incomplete counts for a ticket
+	 */
+	public function get_available_incomplete_counts_for_ticket( $ticket_sold ) {
+
+		$availability = array();
+		if (  $ticket_sold['ticket']->available() > 0 ) {
+			$availability['available'] = sprintf( '%s %s',
+				esc_html( $ticket_sold['ticket']->available() ),
+				esc_html__( 'available', 'event-tickets-plus' )
+			);
+		}
+		if (  $ticket_sold['incomplete'] > 0 ) {
+			$availability['incomplete'] = sprintf( '%s %s',
+				 $ticket_sold['incomplete'],
+				 _n( 'incomplete order', 'incompleted orders', $ticket_sold['incomplete'], 'event-tickets-plus' )
+			);
+		}
+
+		if ( empty( $availability ) ) {
+			return false;
+		}
+
+		return '<div>' . implode( ', ', array_map( 'esc_html', $availability ) ) . '</div>';
+
+	}
+
+	/**
+	 * Get Sales By Ticket Tooltip
+	 *
+	 * @since tbd
+	 *
+	 * @return string a string of html for the tooltip
+	 */
+	public function get_sale_by_ticket_tooltip() {
+		ob_start();
+		?>
+		<div class="tribe-tooltip" aria-expanded="false">
+			<span class="dashicons dashicons-info"></span>
+			<div class="down">
+				<?php echo esc_html__( 'Sold counts completed orders only.', 'event-tickets-plus' ); ?><i></i>
+			</div>
+		</div>
+		<?php
+		return ob_get_clean();
+	}
+
+	/**
+	 * Get Total Sales Tooltip
+	 *
+	 * @since tbd
+	 *
+	 * @return string a string of html for the tooltip
+	 */
+	public function get_total_sale_tooltip() {
+		ob_start();
+		?>
+		<div class="tribe-tooltip" aria-expanded="false">
+			<span class="dashicons dashicons-info"></span>
+			<div class="down">
+				<?php echo esc_html__( 'Total Sales counts all completed orders.', 'event-tickets-plus' ); ?><i></i>
+			</div>
+		</div>
+		<?php
+		return ob_get_clean();
+	}
+
+	/**
+	 * Get Order Tooltip
+	 *
+	 * @since tbd
+	 *
+	 * @return string a string of html for the tooltip
+	 */
+	public function get_total_order_tooltip() {
+		ob_start();
+		?>
+		<div class="tribe-tooltip" aria-expanded="false">
```

Comment:
Any reason these types of functions aren't using views instead?

Reply:
Such as an admin view?

Reply:
Right, just curious why all the markup is in the class instead of views/admin views.